qTox fails to compile

```
In file included from /usr/include/glib-2.0/glib/galloca.h:32:0,
                 from /usr/include/glib-2.0/glib.h:30,
                 from src/platform/statusnotifier/statusnotifier.h:26,
                 from src/platform/statusnotifier/enums.h:3,
                 from src/platform/statusnotifier/enums.c:4:
/usr/include/glib-2.0/glib/gtypes.h:32:24: fatal error: glibconfig.h: No such file or 
directory
 #include <glibconfig.h>
                        ^
In file included from /usr/include/glib-2.0/glib/galloca.h:32:0,
                 from /usr/include/glib-2.0/glib.h:30,
                 from /usr/include/glib-2.0/gobject/gbinding.h:28,
                 from /usr/include/glib-2.0/glib-object.h:23,
                 from src/platform/statusnotifier/closures.c:23:
/usr/include/glib-2.0/glib/gtypes.h:32:24: fatal error: glibconfig.h: No such file or directory
 #include <glibconfig.h>
                        ^
compilation terminated.
compilation terminated.
Makefile:2183: recipe for target 'closures.o' failed
```

--- a/qtox.pro
+++ b/qtox.pro
@@ -206,14 +206,17 @@ contains(ENABLE_SYSTRAY_STATUSNOTIFIER_BACKEND, NO) {
 
 	INCLUDEPATH += "/usr/include/gtk-2.0"
 	INCLUDEPATH += "/usr/include/glib-2.0"
+	INCLUDEPATH += "/usr/lib/glib-2.0/include"
 	INCLUDEPATH += "/usr/lib/x86_64-linux-gnu/glib-2.0/include"
 	INCLUDEPATH += "/usr/lib/i386-linux-gnu/glib-2.0/include"
+	INCLUDEPATH += "/usr/lib/gtk-2.0/include"
 	INCLUDEPATH += "/usr/lib/x86_64-linux-gnu/gtk-2.0/include"
 	INCLUDEPATH += "/usr/lib/i386-linux-gnu/gtk-2.0/include"
+	INCLUDEPATH += "/usr/include/atk-1.0"
 	INCLUDEPATH += "/usr/include/gdk-pixbuf-2.0"
 	INCLUDEPATH += "/usr/include/cairo"
 	INCLUDEPATH += "/usr/include/pango-1.0"
-	INCLUDEPATH += "/usr/include/atk-1.0"
+
 
 	LIBS += -lglib-2.0 -lgdk_pixbuf-2.0 -lgio-2.0 -lcairo -lgtk-x11-2.0 -lgdk-x11-2.0 -lgobject-2.0
 

